### ISSUE TITLE
Error while running main_cmd.py

### ISSUE THREAD

**Issue by rabiaali95** [christian-rncl/TrackNPred#5]
I have the resources folder downloaded using 'python3 setup.py'. Now when i run the command  'python3 main_cmd.py --list 13 --detection False --tracking False --train True --eval True' i have an error
 "DeprecationWarning: The linear_assignment_ module is deprecated in 0.21 and will be removed from 0.23. Use scipy.optimize.linear_sum_assignment instead.
  DeprecationWarning)
usage: main_cmd.py [-h] [--dataset_name DATASET_NAME] [--delim DELIM]
                   [--loader_num_workers LOADER_NUM_WORKERS]
                   [--obs_len OBS_LEN] [--pred_len PRED_LEN] [--skip SKIP]
                   [--batch_size BATCH_SIZE] [--num_iterations NUM_ITERATIONS]
                   [--num_epochs NUM_EPOCHS] [--embedding_dim EMBEDDING_DIM]
                   [--num_layers NUM_LAYERS] [--dropout DROPOUT]
                   [--batch_norm BATCH_NORM] [--mlp_dim MLP_DIM]
                   [--encoder_h_dim_g ENCODER_H_DIM_G]
                   [--decoder_h_dim_g DECODER_H_DIM_G] [--noise_dim NOISE_DIM]
                   [--noise_type NOISE_TYPE] [--noise_mix_type NOISE_MIX_TYPE]
                   [--clipping_threshold_g CLIPPING_THRESHOLD_G]
                   [--g_learning_rate G_LEARNING_RATE] [--g_steps G_STEPS]
                   [--pooling_type POOLING_TYPE]
                   [--pool_every_timestep POOL_EVERY_TIMESTEP]
                   [--bottleneck_dim BOTTLENECK_DIM]
                   [--neighborhood_size NEIGHBORHOOD_SIZE]
                   [--grid_size GRID_SIZE] [--d_type D_TYPE]
                   [--encoder_h_dim_d ENCODER_H_DIM_D]
                   [--d_learning_rate D_LEARNING_RATE] [--d_steps D_STEPS]
                   [--clipping_threshold_d CLIPPING_THRESHOLD_D]
                   [--l2_loss_weight L2_LOSS_WEIGHT] [--best_k BEST_K]
                   [--output_dir OUTPUT_DIR] [--print_every PRINT_EVERY]
                   [--checkpoint_every CHECKPOINT_EVERY]
                   [--checkpoint_name CHECKPOINT_NAME]
                   [--checkpoint_start_from CHECKPOINT_START_FROM]
                   [--restore_from_checkpoint RESTORE_FROM_CHECKPOINT]
                   [--num_samples_check NUM_SAMPLES_CHECK] [--use_gpu USE_GPU]
                   [--timing TIMING] [--gpu_num GPU_NUM]
                   [--model_path MODEL_PATH] [--num_samples NUM_SAMPLES]
                   [--dset_type DSET_TYPE]
main_cmd.py: error: unrecognized arguments: --list 13 --detection False --tracking False --train True --eval True". 
How to get rid of this error?

**Comment by rayguan97** [comment]
Thanks for the comment! Now I updated the code and you should be able to use the command "python3 main_cmd.py --list 13 --detection False --tracking False --train True --eval True". Sorry about the inconvenience!

**Comment by rabiaali95** [comment]
Thank you for solving this. There is one more error that i am facing with. When i run main.py with my own video and corresponding frames i get the error as shown in the image. How can i solve this?
![error](https://user-images.githubusercontent.com/31028173/63916904-586bfd00-ca74-11e9-810e-68ddc2a1859c.png)

**Comment by rabiaali95** [comment]
The error is gone. Thank you
